generators/client/templates/react/src/main/webapp/app/shared/reducers/authentication.ts.ejs

```diff
@@ -226,7 +226,7 @@ export const logout = () => async dispatch => {
   });
 
   // fetch new csrf token
-  dispatch(getSession());
+  // dispatch(getSession());
```

Comment:
Should this remain commented out?

Reply:
@JasonTypesCodes I'm unclear as to what that line of code is supposed to do. It appears to trigger a query to get the user account, but it always fails with a 401 as the user has already been logged out.